Setup/Model/ProductInstaller.php

```diff
@@ -101,7 +92,6 @@ public function createProduct($attributeSet)
             if ($product = $this->createProtectionPlanProduct($attributeSet)) {
                 $this->addImageToPubMedia();
                 $this->processMediaGalleryEntry($this->getMediaImagePath(), $product->getSku());
-                $this->createSourceItem();
```

Comment:
When testing with the existing code (i.e. `createSourceItem` called) and the new version (removed creation of source item) the `inventory_source_item` table resulted with the following identical entry:
![Screen Shot 2023-08-02 at 12 21 26 PM](https://github.com/helloextend/magento-extend-integration/assets/6566341/7d50762c-39a6-490d-ac2a-c0520498dba8)